[CI] MappingSupplementaryCharacterRollingUpgradeIT testMappingWithSupplementaryCharacterFieldName {upgradedNodes=2} failing

**Build Scans:**
- [elasticsearch-intake #35896 / 9.2.7_bwc-snapshots](https://gradle-enterprise.elastic.co/s/omy3ibrqbr3xy)
- [elasticsearch-intake #35894 / 9.3.2_bwc-snapshots](https://gradle-enterprise.elastic.co/s/wthqwmey4k7uq)
- [elasticsearch-intake #35894 / 8.19.13_bwc-snapshots](https://gradle-enterprise.elastic.co/s/24hty7oteqv54)
- [elasticsearch-pull-request #128095 / 9.3.2_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/xmepnrjs6rnve)
- [elasticsearch-pull-request #128081 / 9.2.7_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/6prhphqowgkww)
- [elasticsearch-pull-request #128081 / 9.2.7_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/qmtcz2sowzuty)
- [elasticsearch-pull-request #128009 / 9.3.2_bwc](https://gradle-enterprise.elastic.co/s/k4d6v2olrpwuo)
- [elasticsearch-pull-request #128009 / 8.19.13_bwc](https://gradle-enterprise.elastic.co/s/m6fymwilnckam)
- [elasticsearch-pull-request #127901 / 8.19.13_bwc](https://gradle-enterprise.elastic.co/s/3653st4atjr5w)
- [elasticsearch-pull-request #127901 / 9.0.8_bwc](https://gradle-enterprise.elastic.co/s/iwe54phomfkoe)

**Reproduction Line:**
```
./gradlew ":qa:rolling-upgrade:v9.2.7#bwcTest" -Dtests.class="org.elasticsearch.upgrades.MappingSupplementaryCharacterRollingUpgradeIT" -Dtests.method="testMappingWithSupplementaryCharacterFieldName {upgradedNodes=2}" -Dtests.seed=6A85F487385755B1 -Dtests.bwc=true -Dtests.locale=oc-Latn-FR -Dtests.timezone=Cuba -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.upgrades.MappingSupplementaryCharacterRollingUpgradeIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testMappingWithSupplementaryCharacterFieldName%20%7BupgradedNodes%3D2%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: timed out waiting for green state for index [supplementary_char_mixed] cluster state [{
  "cluster_name" : "test-cluster",
  "cluster_uuid" : "wrqLafmNRoqUMWfcNtFtig",
  "version" : 32,
  "state_uuid" : "oVLanG5ESRK35PDCV1Ju9A",
  "master_node" : "th83wR86Qs-gj6IPmwiegQ",
  "blocks" : { },
  "nodes" : {
    "40Xr71DiSjiRaofpt7NY8Q" : {
      "name" : "test-cluster-1",
      "ephemeral_id" : "JZQ7N9neRvKo-5UXLw_wmQ",
      "transport_address" : "127.0.0.1:37595",
      "external_id" : "test-cluster-1",
      "attributes" : {
        "transform.config_version" : "10.0.0",
        "xpack.installed" : "true",
        "ml.allocated_processors" : "2",
        "ml.max_jvm_size" : "805306368",
        "ml.config_version" : "12.0.0",
        "ml.machine_memory" : "101236109312",
        "ml.allocated_processors_double" : "2.0",
        "testattr" : "test"
      },
      "roles" : [
        "data",
        "data_cold",
        "data_content",
        "data_frozen",
    
[truncated]
```

**Issue Reasons:**
- [main] 2 consecutive failures in test testMappingWithSupplementaryCharacterFieldName {upgradedNodes=2}
- [main] 2 consecutive failures in pipeline elasticsearch-intake
- [main] 10 failures in test testMappingWithSupplementaryCharacterFieldName {upgradedNodes=2} (15.4% fail rate in 65 executions)
- [main] 2 failures in step 9.2.7_part_6_bwc-snapshots (40.0% fail rate in 5 executions)
- [main] 2 failures in step 8.19.13_bwc (50.0% fail rate in 4 executions)
- [main] 2 failures in pipeline elasticsearch-intake (50.0% fail rate in 4 executions)
- [main] 4 failures in pipeline elasticsearch-pull-request (50.0% fail rate in 8 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -294,15 +294,6 @@ tests:
 - class: org.elasticsearch.repositories.azure.AzureBlobContainerRetriesTests
   method: testWriteBlobWithRetries
   issue: https://github.com/elastic/elasticsearch/issues/144025
-- class: org.elasticsearch.upgrades.MappingSupplementaryCharacterRollingUpgradeIT
-  method: testMappingWithSupplementaryCharacterFieldName {upgradedNodes=3}
-  issue: https://github.com/elastic/elasticsearch/issues/144163
-- class: org.elasticsearch.upgrades.MappingSupplementaryCharacterRollingUpgradeIT
-  method: testMappingWithSupplementaryCharacterFieldName {upgradedNodes=1}
-  issue: https://github.com/elastic/elasticsearch/issues/144164
-- class: org.elasticsearch.upgrades.MappingSupplementaryCharacterRollingUpgradeIT
-  method: testMappingWithSupplementaryCharacterFieldName {upgradedNodes=2}
-  issue: https://github.com/elastic/elasticsearch/issues/144165
 - class: org.elasticsearch.test.apmintegration.MetricsApmIT
   method: testJvmMetrics {withOTel=true}
   issue: https://github.com/elastic/elasticsearch/issues/144166

--- a/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/MappingSupplementaryCharacterRollingUpgradeIT.java
+++ b/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/MappingSupplementaryCharacterRollingUpgradeIT.java
@@ -102,7 +102,7 @@ public class MappingSupplementaryCharacterRollingUpgradeIT extends AbstractRolli
             .startObject()
             .startObject("settings")
             .field("number_of_shards", 1)
-            .field("number_of_replicas", 1)
+            .field("number_of_replicas", 0)
             .endObject()
             .startObject("mappings")
             .startObject("properties")

